repctl/pkg/cmd/templates/vxflexos_source.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ .Name }}
+provisioner: csi-{{ .Driver }}.dellemc.com
+reclaimPolicy: {{ .ReclaimPolicy }}
+volumeBindingMode: Immediate
+allowVolumeExpansion: true
+parameters:
+  storagepool: {{ .Parameters.StoragePool.Source }}
+  systemID: {{ .Parameters.RemoteSystem.Source }}
+  {{ .ReplicationPrefix }}/isReplicationEnabled: "true"
+  {{- if eq .TargetClusterID .SourceClusterID }}
+  {{ .ReplicationPrefix }}/remoteStorageClassName: {{ .Name }}-tgt
+  {{ .ReplicationPrefix }}/remoteClusterID: self
+  {{- else }}
+  {{ .ReplicationPrefix }}/remoteStorageClassName: {{ .Name }}
+  {{ .ReplicationPrefix }}/remoteClusterID: {{ .TargetClusterID }}
+  {{- end }}
+  {{ .ReplicationPrefix }}/remoteSystem: {{ .Parameters.RemoteSystem.Target }}
+  {{ .ReplicationPrefix }}/remoteStoragePool: {{ .Parameters.StoragePool.Target }}
+  {{ .ReplicationPrefix }}/rpo: "{{ .Parameters.Rpo }}"
+  {{ .ReplicationPrefix }}/ignoreNamespaces: "{{ .Parameters.IgnoreNamespaces }}"
+  {{ .ReplicationPrefix }}/volumeGroupPrefix: {{ .Parameters.VolumeGroupPrefix }}
+  {{ .ReplicationPrefix }}/remoteRGRetentionPolicy: {{ .RemoteRetentionPolicy.RG }}
+  {{ .ReplicationPrefix }}/remotePVRetentionPolicy: {{ .RemoteRetentionPolicy.PV }}
+  {{ .ReplicationPrefix }}/consistencyGroupName: {{ .Parameters.ConsistencyGroupName }}
```

Comment:
I don't see ConsistencyGroupName in repctl/examples/vxflexos_example_values.yaml

Reply:
Fixed in other PR.